xbmc/cores/AudioEngine/Sinks/pipewire/AESinkPipewire.cpp

```diff
@@ -348,7 +350,8 @@ bool CAESinkPipewire::Initialize(AEAudioFormat& format, std::string& device)
 
   m_latency = DEFAULT_BUFFER_DURATION;
   uint32_t frames = std::nearbyint(DEFAULT_PERIOD_DURATION.count() * format.m_sampleRate);
-  std::string fraction = StringUtils::Format("{}/{}", frames, format.m_sampleRate);
+  std::string fraction =
+      StringUtils::Format("{}/{}", frames / DEFAULT_LATENCY_DIVIDER, format.m_sampleRate);
```

Comment:
Here I think a code comment would be useful, on why the "node" delay by itself needs to be more than twice as fast as the period. Future maintainers won't know without.

Reply:
Yea, I'm trying to find the words to describe it. It would be nice if I could reference some documentation.

Reply:
Maybe add just the short comment: "Node Timing is abstracted from the sink. It needs to be fast enough, so the real sink does not underrun. Therefore let's be more than twice as fast as the periods we transmit" <- in proper english maybe :-) and after that - in with it!

Reply:
ok, I actually added it to the commit message in https://github.com/xbmc/xbmc/pull/22560/commits/ab565800acc7e38d8258704f34ec7e74a2ae17e9

I think it would get lost as an inline comment.

Reply:
From developer POV, when you start reading the code, you read the source file and only if you have doubts, you start git blame. I agree that commit messages for Changes that are cherry-picked or for readability is very important, but having it in the code also is. Never mind -> I for my part won't forget it.